src/main/java/com/iridium/iridiumskyblock/commands/JoinCommand.java

```diff
@@ -64,6 +65,11 @@ public boolean execute(CommandSender sender, String[] args) {
                         return false;
                     }
 
+                    Optional<User> inviter = islandInvite.map(IslandInvite::getInviter);
+                    UserJoinEvent userJoinEvent = new UserJoinEvent(island.get(), user, inviter.orElse(null));
+                    Bukkit.getPluginManager().callEvent(userJoinEvent);
+                    if (userJoinEvent.isCancelled()) return false;
```

Comment:
@dlsf is should be true?
So, command is executed success but just ignored

Reply:
> is should return true?

Oh, it's true, sorry, one minute :)

Reply:
No, technically it was not successful. I'd suggest to return false